downstream/modules/platform/proc-aap-migration-backup.adoc

```diff
@@ -1,6 +1,20 @@
 [id="aap-migration-backup"]
-
 [role="_abstract"]
+
+.Prerequisites
+
+To migrate {PlatformNameShort} deployment to {OperatorPlatform}, you must have the following:
+
+* Secret key secret
+* Postgresql configuration
+* Role-based Access Control for the namespaces on the new OpenShift cluster
+* The new OpenShift cluster must be able to connect to the previous PostgreSQL database
+
+[NOTE]
+====
+Secret key information can be located in the inventory file created during the initial {PlatformName} installation. If you are unable to remember your secret key or have trouble locating your inventory file, contact link:https://access.redhat.com/[Ansible support] via the Red Hat Customer portal.
```

Comment:
```suggestion
You can store the secret key information in the inventory file before the initial {PlatformName} installation. If you are unable to remember your secret key or have trouble locating your inventory file, contact link:https://access.redhat.com/[Ansible support] via the Red Hat Customer portal.
```

I think an active verb would work better here, but my suggestion mihght not be right